containers/podman-desktop
# chore: process podman system and user configuration

### What does this PR do?

The purpose of this changes request is to provide the complex mechanism to process Podman system and user based configuration. According to the latest changes in Podman [new.go](https://github.com/containers/common/blob/main/pkg/config/new.go) there is a need to implement the same logic in Podman Desktop to process these configuration files.

### Screenshot / video of UI

No screenshots. Changes are invisible from the user perspective.

### What issues does this PR fix or reference?

#7027 

### How to test this PR?

Podman Desktop should take into account Podman system-based and user-based configuration. So, no visual check on the UI. The only thing is to check, that the current PR works the same as https://github.com/containers/podman-desktop/pull/7881 and https://github.com/containers/podman-desktop/pull/7540 intended.
